src/components/views/elements/PollCreateDialog.tsx

```diff
@@ -98,10 +100,31 @@ export default class PollCreateDialog extends ScrollableBaseModal<IProps, IState
         this.matrixClient.sendEvent(
             this.props.room.roomId,
             POLL_START_EVENT_TYPE.name,
-            makePollContent(this.state.question, this.state.options, POLL_KIND_DISCLOSED.name),
-        ).then(() => this.props.onFinished(true)).catch(e => {
-            console.error("Failed to submit poll event:", e);
-            this.setState({ busy: false, canSubmit: true });
+            makePollContent(
+                this.state.question, this.state.options, POLL_KIND_DISCLOSED.name),
```

Comment:
```suggestion
                this.state.question, this.state.options, POLL_KIND_DISCLOSED.name,
            ),
```

this indentation was tripping me out otherwise

Reply:
heck, maybe the params should be one-per-line given they're already not on the initial line